cluster-management/setup/mounting-storage/index.md

```diff
@@ -64,6 +64,46 @@ coreos:
 
 Notice that we're starting both units at the same time and using the power of systemd to work out the dependencies for us. In this case, `var-lib-docker.mount` requires `format-ephemeral.service`, ensuring that our storage will always be formatted before it is mounted. Docker will refuse to start otherwise.
 
+## Creating and Mounting a btrfs Volume File
+
+CoreOS versions 6xx.0.0 and later use ext4 + overlayfs to provide a layered filesystem for the root partition.
+Prior to this, CoreOS 6xx.0.0 and earlier used btrfs for this functionality.
+If you'd like to continue using btrfs on newer CoreOS machines, you can do so with two systemd units: one that creates and formats a btrfs volume file and another that mounts it.
+
+In this example, we are going to mount a new 25GB btrfs volume file to `/var/lib/docker`, and one can verify that docker is using the btrfs storage driver once the docker service has started by executing `sudo docker info`.
```

Comment:
Perhaps explicitly state that this file should be no more than about 85% of the root filesystem size, our images ship with a fairly small root in many cases. On aws for example if you don't change the disk size when provisioning you wind up with an 8GB disk.